src/org/zaproxy/zap/extension/websocket/ZapAddOn.xml

```diff
@@ -1,14 +1,13 @@
 <zapaddon>
 	<name>WebSockets</name>
-	<version>15</version>
+	<version>16</version>
```

Comment:
It should remain in 15 (changes should be kept as well).

Reply:
I thought 15 was already released

Reply:
No problem, I've made that mistake in the past. 

https://github.com/zaproxy/zap-extensions/releases